server/container/guice/cassandra/src/main/java/org/apache/james/modules/data/CassandraJmapModule.java

```diff
@@ -80,7 +79,6 @@ protected void configure() {
         bind(CassandraUploadRepository.class).in(Scopes.SINGLETON);
         bind(UploadDAO.class).in(Scopes.SINGLETON);
         bind(UploadRepository.class).to(CassandraUploadRepository.class);
-        bind(UploadConfiguration.class).toInstance(UploadConfiguration.SINGLETON);
```

Comment:
Modify documentation?

Reply:
No documentation related to upload configuration (it was not added before)